### DISCUSSION TITLE
Theme resets to default after closing.

### DISCUSSION THREAD

**Discussion by blabberlicious** [MCC321-QC/eMixedNiteMC#65] [General]
Love your theme, but if I toggle and editsome view settings via the widgets at the bottom left, they don't stick. Is there a way to lock in the changes I make to the theme? Tried closing the modification Cog Icon, but do dice.

thank you for this. It's perfect, otherwise!

**Comment by CapnMurica** [comment]
I have found a workaround for this. Go to the installation folder for this theme and open the file Constants.xaml in your text editor of choice. From here, modify the values to your pleasure.  They are all pretty aptly named. Once you have modified all the values to you linking you can save the file. Now when you restart Playnite all the settings you made in this file will get applied. It would also be a good idea to make a backup of the xaml file. Hopefully the dev of this theme will add the ability to save the settings you make without needing to modify this file.

**Comment by MCC321-QC** [comment]
Those settings are only "quick view" settings so you can try them and find what you want easier than opening the constant menu for each settings. Those settings are meant to be edited with the plugin [ThemeModifier](https://github.com/Lacro59/playnite-thememodifier-plugin). There is other settings you can change with the extension [Theme Options](https://github.com/ashpynov/ThemeOptions). Check [this](https://github.com/MCC321-QC/eMixedNiteMC/issues/62) for more info, and I would suggest to get a quick look at the [readme](https://github.com/MCC321-QC/eMixedNiteMC?tab=readme-ov-file#how-to-install), there is other important information regarding this theme there😄
